src/main/java/com/example/AllPermutationPrime.java

```diff
@@ -0,0 +1,62 @@
+package com.example;
+
+import java.util.ArrayList;
+
+/**
+ * Készíts egy olyan függvényt, aminek
+ * bemenő paramétere n,
+ * és válaszként kilistázza azokat a számokat 1-től n-ig,
+ * amelyeknek az összes körkörös permutációja prím.
+ */
+public class AllPermutationPrime {
+
+    private static boolean isPrime(int n) {        
+        boolean prime = true;
+        int divider = n - 1;
+        while ((divider > 1) && prime) {
+            if (n % divider == 0) {
+                prime = false;
+            }
+            else {
+                divider--;
+            }
+        }
+        return prime;
+    }
+
+    private static int[] generateCircledPermutations(int checkedNumber) {
+        String number = String.valueOf(checkedNumber);       
+        int numberOfDigits = number.length();
+        number += number;
+        int[] permutations = new int[numberOfDigits];
+        for (int i = 0; i < numberOfDigits; i++) {
+            permutations[i] = Integer.parseInt(number.substring(i, i + numberOfDigits));
+        }
+        return permutations;
+    }
+
+    public static int[] allCircledPermutationsArePrimes(int n) {
+        ArrayList<Integer> goodNumbers = new ArrayList<>();
```

Comment:
I don't think that's the right name for the variable. `good` and `not_good` is debatable and not related to what we try to achieve here. Please try to find a more descriptive name.

Reply:
If it is not a problem, used very descriptive (verbal) namings with note that exact naming results long variable names, due to complexity of the task. E.g. numbersWithAllTheirCircledPermutationsArePrimesArray.